New guide to integrate Jira with Incident Management (#22204)
DataDog/documentation: content/en/service_management/incident_management/guides/jira.md

---
title: Integrate Jira with Datadog Incident Management
kind: Guide
disable_toc: false
further_reading:
- link: "integrations/jira/"
  tag: "Documentation"
  text: "Install the Jira Integration"
- link: "https://app.datadoghq.com/integrations/jira"
  tag: "In-App"
  text: "In-app Jira integration tile"

---

## Overview

> Jira is an issue and project tracking system for software teams. The Datadog Jira integration allows you to create issues incidents in Datadog and view issues created in Jira as Datadog events. Use the Jira integration with Datadog Incident Management for:

Reviewer: ```suggestion
Jira is an issue and project tracking system for software teams. The Datadog Jira integration allows you to create issues incidents in Datadog and view issues created in Jira as Datadog events. 

The Jira integration with Datadog Incident Management provides you with the following benefits:
```

- **Improved Visibility**: Ensure that all stakeholders are immediately informed about incidents, facilitating a quicker response.
- **Supporting Existing Workflows**: Seamlessly integrate with your current processes, making it easier to plan work and manage priorities with Jira.
- **Customization at Your Fingertips**: With dynamic templates, you can map Datadog severities to Jira priorities, add custom labels, define dynamic assignees, and more.

## Prerequisites

To use automatic ticket creation, install the integration through the [Jira Integration tile][1]. For more information, see the [Jira integration][2] documentation.

## Setup

> 1. In the [Integration Settings page][3], find the Statuspage integration. 

Reviewer: ```suggestion
1. On the [Integration Settings page][3], find the Statuspage integration. 
```

> 1. Toggle the **Automatically create a Jira Issue**.

Reviewer: ```suggestion
1. Click the toggle for **Automatically create a Jira Issue**.
```
